src/components/status-bar/StatusBar.vue

```diff
@@ -43,34 +44,44 @@ export default {
   props: {
     portUsageDown: {
       type: Number,
+      default: 0,
     },
     portUsageUp: {
       type: Number,
+      default: 0,
     },
     packetError: {
       type: Number,
+      default: 0,
     },
     i2cError: {
       type: Number,
+      default: 0,
     },
     cycleTime: {
       type: Number,
+      default: 0,
     },
     cpuLoad: {
       type: Number,
+      default: 0,
```

Comment:
Same here and also for the others elements.

Reply:
Same here, defaults here are for the component encapsulation, not for the global variables.